### PR TITLE
Add mkdocs

### PR DESCRIPTION
**Pull request by mingness**
Add how to for mkdocs, skeleton pages, and workflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,23 +78,47 @@ task has bundled the following required tasks:
...
+
+1. Install [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
+   Please refer to the documentation. For example, run `pip install mkdocs-material` in a terminal.
+2. We already created a very barebones website in the template. you can
+   see it in action by running `mkdocs serve`
+3. As seen in the documentation, the configuration of the website
+   is defined in the `mkdocs.yml` file. We have included some plugins, such
+   as search, code block copy and annotations, and navigation bars. Of course,
+   configure your website as you prefer.
+4. The template has a Github workflow for deploying the MkDocs website
```
Comment (Stefterv) [inline README.md:111]: I would put this first

File: README.md [hunk 2]
```diff
@@ -46,23 +78,47 @@ task has bundled the following required tasks:
...
+advantage of Github Pages to deploy and host the website. We configure this template for this scenario.
+
+MkDocs is Python-based, so if you are not already a Python user, hopefully your OS already has a system
+Python installed. You can check by running `python --version` in a terminal. If you do not already have
+Python installed, or do not have Python 3, you will need to install Python 3 as a prerequisite to these
+instructions.
+
+1. Install [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
+   Please refer to the documentation. For example, run `pip install mkdocs-material` in a terminal.
+2. We already created a very barebones website in the template. you can
```
Comment (Stefterv) [inline README.md:105]: Same, let's have the default path be just publishing through editing the repository, then if people want live-preview then can go through installing MKDocs themselves.

File: README.md [hunk 3]
```diff
@@ -46,23 +78,47 @@ task has bundled the following required tasks:
...
+have clued us into [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/),
+as a simple and easy to use way of creating content. If your library is on Github, then you can take
+advantage of Github Pages to deploy and host the website. We configure this template for this scenario.
+
+MkDocs is Python-based, so if you are not already a Python user, hopefully your OS already has a system
+Python installed. You can check by running `python --version` in a terminal. If you do not already have
+Python installed, or do not have Python 3, you will need to install Python 3 as a prerequisite to these
+instructions.
+
+1. Install [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
```
Comment (Stefterv) [inline README.md:103]: Make this optional

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Welcome to MkDocs
+
+For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+
+## Commands
+
+* `mkdocs new [dir-name]` - Create a new project.
```
Comment (Stefterv) [inline docs/index.md:7]: Please make this an example of what a library page would look like.

**Review by Stefterv — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 4 above)
Comment (mingness) [reply docs/index.md]: DONE

**Review by Stefterv — APPROVED** [review]

**Pull request merged**